### PR TITLE
Tickets/dm 40354

### PR DESCRIPTION
**Pull request by edennihy**

### CODE REVIEW

File: python/lsst/ts/standardscripts/auxtel/calibrations/power_off_atcalsys.py [hunk 1]
```diff
@@ -125,9 +126,7 @@ async def wait_for_lamp_to_cool_down(self):
                 lamp_state = await self.white_light_source.evt_lampState.next(
                     flush=False, timeout=self.timeout_lamp_cool_down
                 )
-                cool_down_wait_time = (
-                    lamp_state.cooldownEndTime - lamp_state.private_sndStamp
-                )
+                cool_down_wait_time = lamp_state.cooldownEndTime - time.time()
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/auxtel/calibrations/power_off_atcalsys.py:129]: I think what you want to do is replace `cool_down_wait_time` to `self.timeout_lamp_cool_down` in line 132. This here doesn't really make sense.  Or, actually, just remove line 132.

File: python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py [hunk 2]
```diff
@@ -34,6 +34,8 @@ class PowerOnATCalSys(salobj.BaseScript):
     (ATWhiteLight and ATMonochromator)
     required to perform image calibrations over white light.
 
+    August 13thb;: Commented all ATMonochromator as it's configured and off.
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py:37]: Please, remove this line.

File: python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py [hunk 3]
```diff
@@ -110,6 +112,13 @@ def get_schema(cls):
                 minimum: 0
                 default: 7
 
+              use_atmonochromator:
+                description: Is the monochromator available and can be configured?
+                    If False, the monochromator will be left as it is.
+                    If True, the monochromator will be commanded to the configuration for white light.
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py:118]: Consider:

```
If True, the monochromator will be configured for white light.
```

**Review by tribeiro — COMMENTED** [review]
Looks good but I have some comments I would hope you will consider before I can approve the PR.

File: python/lsst/ts/standardscripts/auxtel/calibrations/power_off_atcalsys.py (hunk 1 above)
Comment (edennihy) [reply python/lsst/ts/standardscripts/auxtel/calibrations/power_off_atcalsys.py]: Removed line 132 and caught a related bug lower down in the exception message.

**Review by tribeiro — APPROVED** [review]
looks good

**Pull request merged**